client/src/functions/calculate-score.ts

```diff
@@ -0,0 +1,34 @@
+import { PieceType } from '../model/piece';

```

Comment:
I deliberately left the calculation in the bot code, since I was planning on providing multiple bots, each with its own calculation method. This will give the bots different "characters", as well as different strenth resulting from the depth. For example, bot1 ("Andy") might like knights more than bishops. bot2 ("Ben") always try to check your king as much as possible. bot3 ("Charlie") likes early queen attacks, etc.